Managers/SceneMan.cpp

```diff
@@ -3788,13 +3602,11 @@ void SceneMan::Draw(BITMAP *pTargetBitmap, BITMAP *pTargetGUIBitmap, const Vecto
 //            m_pMOColorLayer->Draw(pTargetBitmap, targetBox);
```

Comment:
Can you kill off all this commented debugging junk while you're here. Same with below.

Reply:
I kept them because maybe we do want that text to show in different drawing modes?

Reply:
Well if we do want the text you got rid of the frameman drawtext part lol. I'd say try it out and see, you're right that it might not be bad to have showing.

Reply:
I got rid of it because it was a `lel this doesn't work just crashes your game` method and was too lazy to replace it with a working text draw method. kek
Will do soon™.

Reply:
PrimitiveMan DrawText wouldn't work??
Either way, kill the commented code and implement it or put a TODO that you'll never get to telling you to implement it :D

Reply:
`PrimitiveMan` methods only ever deal with primitive drawing via lua, they aren't used anywhere in the cpp.
`FrameMan::DrawText` was just a `DDTAbort("Old fonts are not ported yet!");` so I need to replace it with `GUIFont::DrawAligned`.

Reply:
Ahhhh, gotcha. Good to know. I'll leave it to you to deal with appropriately.